### PR TITLE
LNDC design for themes

### PR DESCRIPTION
**Pull request by nolliia**
image

- fixed themes are now clickable
- centred the footed and the header
- fixed the padding between the footer and the main page
- removed extra padding

## Summary by Sourcery

This pull request enhances the UI by making themes clickable, centering the footer and header, and adjusting padding for better layout consistency. Extra padding has been removed to streamline the design.

- **Enhancements**:
    - Made themes clickable in the ThemesSection component.
    - Centered the footer and header for better alignment.
    - Adjusted padding between the footer and the main page for consistent spacing.
    - Removed extra padding in various components to streamline the layout.

### CODE REVIEW

**Comment by zalborzi** [comment]
Could you please handle sourcery's comment related to performance issues?

File: src/components/ThemesSection.tsx [hunk 1]
```diff
@@ -79,9 +79,9 @@ const ThemesSection = ({ maxThemes }: { maxThemes?: number }) => {
           <a
             key={theme.name}
             className="bg-white py-6 flex items-center justify-center px-2 rounded-lg h-[166px] shadow-lg border-b-4 border-b-[#B5BFC4] hover:border-b-secondary transition hover:bg-gray-50"
-            href={`/datasets?group=${theme.name
+            href={`/datasets?page=1&ckan-theme=${(theme.desc || theme.name)
```
Comment (brunopacheco1) [inline src/components/ThemesSection.tsx:82]: important: Can you please reuse a function to support decoding text to URL? In JavaScript there a few ways to do that.

**Review by brunopacheco1 — CHANGES_REQUESTED** [review]

File: src/components/ThemesSection.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (nolliia) [reply src/components/ThemesSection.tsx:82]: I was going to have a discussion about that with you, we can merge this and will add the URI in another pr if you don't mind

Comment (nolliia) [reply src/components/ThemesSection.tsx:82]: I will do them in the next pr when I add the URI's

[the remark this answers is not in the document]
Comment (brunopacheco1) [reply src/components/ThemesSection.tsx:82]: https://publications.europa.eu/resource/authority/data-theme

**Review by brunopacheco1 — APPROVED** [review]

**Pull request merged**